charts/logan/templates/fluentd-deployment.yaml

```diff
@@ -84,6 +86,53 @@ spec:
         {{- if .Values.extraVolumeMounts }}
         {{- toYaml .Values.extraVolumeMounts | nindent 8 }}
         {{- end }}
+      {{- if $enableEKSCPLogs }}
```

Comment:
We are going to remove this Deployment in a weeks time as we are moving Object Logs collection to Discovery flow. Let's discuss on this once.